keyboards/dztech/dz60rgb/keymaps/piv3rt/keymap.c

```diff
@@ -0,0 +1,232 @@
+        _______, _______,  _______,                            RGB_RST,                   _______, _______,  KC_PWR,   AMDREC,  AMDREP
+    ),
+    [_NUM] = LAYOUT(
+        KC_NLCK, _______,  _______, _______, _______, _______, _______, _______, KC_P7,   KC_P8,   KC_P9,    _______,  _______, _______,
+        _______, _______,  _______, _______, _______, _______, _______, _______, KC_P4,   KC_P5,   KC_P6,    _______,  _______, _______,
+        _______, _______,  _______, _______, _______, _______, _______, _______, KC_P1,   KC_P2,   KC_P3,    KC_PENT,           KC_PENT,
+        _______,           _______, _______, _______, _______, _______, _______, _______, KC_P0,   KC_PDOT,  TG(_NUM), _______, _______,
+        _______, _______,  _______,                            KC_PENT,                   _______, _______,  _______,  _______, _______
+    ),
+};
+
+void reset_leds(void) {
+    current_profile = OFF;
+    rgblight_enable();
+    rgblight_mode(RGBLIGHT_MODE_STATIC_LIGHT);
+    rgb_matrix_sethsv(0x6a, 0x33, 0xff);
+}
+
+void suspend_wakeup_init_user(void) {
+    current_profile = OFF;
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+    case RGB_RST:
+        if (record->event.pressed) {
+            reset_leds();
+        }
+        return false;
+        break;
+    case RGB_PCY:
+        if (record->event.pressed) {
+            // Cycle through RGB profiles
+            current_profile = current_profile == OFF ? 0 : current_profile + 1;
+        }
+        return false;
+        break;
+    case RGB_000:
+        if (record->event.pressed) {
+            rgb_matrix_sethsv(0, 0, 0);
+        }
+        return false;
+        break;
+    case EACUTE:
+        if (record->event.pressed) {
+            SEND_STRING(SS_DOWN(X_LALT) SS_TAP(X_P0) SS_TAP(X_P2) SS_TAP(X_P0) SS_TAP(X_P1) SS_UP(X_LALT));
+        }
+        return false;
+        break;
+    case CCED:
+        if (record->event.pressed) {
+            SEND_STRING(SS_DOWN(X_LALT) SS_TAP(X_P0) SS_TAP(X_P1) SS_TAP(X_P9) SS_TAP(X_P9) SS_UP(X_LALT));
+        }
+        return false;
+        break;
+    case AGRAVE:
+        if (record->event.pressed) {
+            SEND_STRING(SS_DOWN(X_LALT) SS_TAP(X_P0) SS_TAP(X_P1) SS_TAP(X_P9) SS_TAP(X_P2) SS_UP(X_LALT));
+        }
+        return false;
+        break;
+    case FRQLT:
+        if (record->event.pressed) {
+            SEND_STRING(SS_DOWN(X_LALT) SS_TAP(X_P1) SS_TAP(X_P7) SS_TAP(X_P4) SS_UP(X_LALT));
+        }
+        return false;
+        break;
+    case FRQGT:
+        if (record->event.pressed) {
+            SEND_STRING(SS_DOWN(X_LALT) SS_TAP(X_P1) SS_TAP(X_P7) SS_TAP(X_P5) SS_UP(X_LALT));
+        }
+        return false;
+        break;
+    }
+    return true;
+};
+
+void rgb_matrix_indicators_advanced_user(uint8_t led_min, uint8_t led_max) {
+    led_t led_state = host_keyboard_led_state();
+
+    if (!g_suspend_state && rgb_matrix_config.enable) {
```

Comment:
This isn't needed anymore. Indicator functions only run if the LEDs are enabled.

```suggestion
```

Reply:
Done 👍